### DISCUSSION TITLE
Error whe tried to run the Win form

### DISCUSSION THREAD

**Discussion by juniosa123** [cyanfish/naps2#195] [Development]
Hi everyone,

I have blocker whe I tried to run this system. The error information like below:

![image](https://github.com/cyanfish/naps2/assets/87955114/aad721ac-c22a-4389-9aa2-7b8f2564d23e)

Please help me. Thank you

**Comment by cyanfish** [comment]
Have a look at the wiki for build instructions:
https://github.com/cyanfish/naps2/wiki/1.-Building-&-Development-Environment

Specifically you'll need to run `dotnet workload install macos` from the terminal
